core-bundle/contao/themes/flexible/styles/basic.css

```
@@ -347,10 +347,31 @@ table.with-zebra tbody tr:nth-child(even) td {
 }
 
 /* Widgets */
-.widget {
-	margin-left: 15px;
-	margin-right: 15px;
-	position: relative;
+fieldset > .widget-group {
+	padding: 0 15px;
+}
+.widget-group {
+	display: grid;
+	grid-column-gap: 30px;
+	grid-template-columns: repeat(12, 1fr);
```

```suggestion
	grid-template-columns: repeat(12, minmax(0, 1fr));
```

Otherwise items that are too wide would blow up the column width.